### ISSUE TITLE
Validation set for the early stopping 

### ISSUE THREAD

**Issue by cuicathy** [ninatu/anomaly_detection#4]
Hi, 

Thanks for your great work! I have a question about the early stopping. According to your paper, 200 tumor patches (abnormal samples) were used as the validation set. In your code, I noticed that the sum of the adversarial loss and reconstruction loss of the validation set was used as the sign for early stopping. I am a little confused why the loss score of abnormal samples in the validation set instead of the AUC score of validation sets with both abnormal and normal images available is used for early stopping. 

I am looking forward to your reply. Thank you!

**Comment by ninatu** [comment]
Hi!

Thanks for the question. We used loss on the validation set for earlier stopping to avoid overfitting on the train data (with the given set hyperparams). And then, we use ROC AUC to choose the best hyperparameters. However, you are right,  ROC AUC  may also be used for earlier stopping.
